### PR TITLE
Update 13-4.cgel

### PR DESCRIPTION
**Pull request by mebwells**

### CODE REVIEW

**Comment by bwaldon** [comment]
- Lowercase “T” in “thus” 
- Agent by-PP “by the court” should be Comp, not Mod (see, e.g., `reviews-351727-0002` in https://github.com/nert-nlp/cgel/blob/main/datasets/ewt.cgel). 
- Add lemma and POS tags in the .cgel file
- I agree with `Mod` label for “to conduct the election” (analogous case: `reviews-371300-0007` in https://github.com/nert-nlp/cgel/blob/main/datasets/ewt.cgel)

File: datasets/oneoff/13-4.cgel [hunk 1]
```diff
@@ -34,7 +34,7 @@
             :Subj (NP
                 :Det (NP
                     :Head (Nom
-                        :Head (N_pro :t "its")))
+                        :Head (N_pro :t "its" :l "it")))
```
Comment (nschneid) [inline datasets/oneoff/13-4.cgel:37]: I don't think we have been using `:l` for pronouns (you can double-check in CGELBank)

Comment (mebwells) [reply datasets/oneoff/13-4.cgel:37]: I've been seeing :l for only possessive pronouns in the twitter examples, but I can stop using it if that would be preferable

Comment (nschneid) [reply datasets/oneoff/13-4.cgel:37]: Ah, it's inconsistent in CGELBank so either way is OK for now. I've opened an issue for making it consistent: nert-nlp/cgel/issues/128

**Comment by bwaldon** [comment]
For more on by-PP complements in nominals, section 5.4.1 of SIEG (p. 128):

Screenshot 2024-09-03 at 5 28 43 AM

**Comment by nschneid** [comment]
What's the rationale for the high attachment of the purpose clause modifier ("meeting to conduct...")?

**Comment by bwaldon** [comment]
@mebwells will open a new PR on the tree
@bwaldon will also make explicit in the guidelines: we have a default preference for low *right* attachment.

**Comment by bwaldon** [comment]
https://github.com/nert-nlp/legal-cgel/pull/49

**Pull request merged**